dataprofiler/profilers/profile_builder.py

```diff
@@ -504,7 +511,33 @@ def _update_row_statistics(self, data):
         self.hashed_row_dict = dict.fromkeys(
             pd.util.hash_pandas_object(data, index=False), True
         )
-        self.null_in_row_count = data.isnull().any(axis=1).sum()
+
+        # Calculate Null Column Count
+        null_rows = {}
+        null_in_row_count = {}
+        first_col_flag = True
+        for column in self.profile:
+            if "statistics" in self.profile[column].profile and \
+                    "null_types_index" in self.profile[column]\
+                    .profile["statistics"]:
+
+                # Cycle through each column and get all the null indices 
+                null_type_dict = self.profile[column].profile["statistics"]\
+                                 ["null_types_index"]
+                null_row_indices = {item for sublist in null_type_dict.values()
+                                    for item in sublist}
```

Comment:
now i'm not sure about optimization, but this in of itself is creating a set.

Is it best to conduct the set operation on all these smaller lists and then a final operation merging all sets, or is it better to combine as a list and perform a single set operation?

Just thoughts